articles/add-a-subdomain.mdx

```diff
@@ -0,0 +1,64 @@
+---
+_schema: default
+_uuid: a0b1ee01-42aa-4412-8991-bca3f4e3793c
+_created_at: 2024-11-29T11:59:04+13:00
+title: Add a subdomain
+nav_title: Add a subdomain
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: >-
+  Learn how to add a subdomain to your Site on CloudCannon and host your website
+  on a subdomain as your primary URL.
+article_category:
+  - Instructions
+article_topic:
+  - Custom Domains
+tags: []
+related_articles: []
+related_links: []
+explicit_canonical:
+---
+A [subdomain](/documentation/articles/add-a-subdomain/) is an additional name on the front of your URL. Subdomains help you to separate and organize content on your Site.
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/hosting/what-is-web-hosting/2024-11/CloudCannon-Documentation-URL-Structure-Diagram.png" alt="A diagram of a URL shows labels for protocol, subdomain, domain, top-level domain, subdirectory, slug, and path." title="The structure of a URL" type="photo"/>
+
+To connect a subdomain, you must have access to the main domain name.
+
+To add a custom subdomain to a Site in CloudCannon:
+
+1. Open the Site to which you want to add the subdomain.
+2. Navigate to the *Custom Domain* page under *Site Settings*.
+3. Select the *Add new custom domain* option in the *Choose your hosting option* dropdown.
+4. In the *Domain Name* text field, enter the subdomain, domain, and tld of you want to connect to this Site (e.g., support.example.com).
+5. Click the *Add Domain* button.
+
+CloudCannon will connect your subdomain to your Site.
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/hosting/add-a-custom-domain/2024-11/CloudCannon-Documentation-Add-Custom-Domain-Site-Settings.png" alt="A screenshot of the Custom Domain page under Site Settings shows the Choose your hosting option dropdown and a Domain name text field." title="The Custom Domain page" type="screenshot"/>
+
+Subdomains do not get a dedicated Card in the *Domains browser*. Instead, your Site will appear in the list of Sites using the main domain name (i.e., the Site connected to support.example.com will appear in the list of Sites connected to example.com).
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/hosting/add-a-custom-domain/2024-11/CloudCannon-Documentation-Domains-Browser-Page.png" alt="A screenshot of the Domain Browser shows four Domain cards with metadata for connected Sites, DNS, and SSL certificate expiry date." title="The Domain Browser" type="screenshot"/>
+
+To stop hosting your Site on a subdomain, navigate to the *Danger zone* tab on the *Custom Domain* page and click the *Remove Custom Domain* button.
+
+## Host your Site on a subdomain
+
+If you want your main domain to be a subdomain (i.e., you do not want users visiting a URL with no subdomain), you need to redirect your domain name to the subdomain.
```

Comment:
`i.e., you do not want users visiting a URL with no subdomain` this is not exactly the behaviour of what you get when changing your `main domain` setting.